language/mandelbrot.toit

```diff
@@ -0,0 +1,53 @@
+/**
+Mandelbrot example in Toit.
+
+This example draws a shape as the output.
+
+*/
+
+MAX_PIXEL_VALUE ::= 255
+
+main:
+  mandelbrot 200 80 -0.6 0.0 0.015 500
+  mandelbrot 200 80 -0.743030 0.126433 0.00003 5000
+
+
+mandelbrot width height x_center y_center scale limit:
+  pixels := List width:
+    ByteArray height
```

Comment:
I would put it on the previous line, but ok like this.

Reply:
For readability, I will put it on the previous line.